src/Serialization/Fields.jl

```diff
@@ -48,3 +48,132 @@ function load_internal(s::DeserializerState, ::Type{gfp_fmpz_elem}, dict::Dict)
     F = load_type_dispatch(s, Nemo.GaloisFmpzField, dict[:parent])
     return F(load_type_dispatch(s, fmpz, dict[:data]))
 end
+
+################################################################################
+# SimpleNumField
+function save_internal(s::SerializerState, K::SimpleNumField)
+    return Dict(
+        :def_pol => save_type_dispatch(s, defining_polynomial(K)),
+        :var => save_type_dispatch(s, String(var(K)))
+    )
+end
+
+function load_internal(s::DeserializerState, ::Type{<: SimpleNumField}, dict::Dict)
+    def_pol = load_type_dispatch(s, dict[:def_pol], check_namespace=false)
+    var = load_type_dispatch(s, dict[:var], check_namespace=false)
```

Comment:
We know the type here, right? So how about this: (alternatively, serialize it as a symbol? provided the NumberField already supports it)
```suggestion
    var = load_type_dispatch(s, String, dict[:var])
```

Reply:
https://github.com/thofma/Hecke.jl/pull/721